### ISSUE TITLE
Sidebar: bottom slider not working as expected in an example

### ISSUE THREAD

**Issue by sansarip** [Semantic-Org/Semantic-UI-React#3884]
### Steps

1. Navigate to the [Slider transitions example under Examples](https://react.semantic-ui.com/modules/sidebar/#examples-transitions).
2. Change the Direction to Bottom by simply clicking the button labeled Bottom
3. Examine the application content view below the configuration buttons and note that you're able to scroll down in that view i.e. there is extra space below the pusher content.

### Expected Result
Given a pushable parent, when a pusher and bottom-directional slider are added to the parent pushable, then the bottom slider should not cause extra space to be rendered below the pusher content

### Actual Result
Extra space is rendered below the pusher content when a bottom-directional slider is present.

### Version
0.88.2

### Testcase
https://react.semantic-ui.com/modules/sidebar/#examples-transitions

**Comment by layershifter** [comment]
@sansarip thanks for reporting, I opened https://github.com/fomantic/Fomantic-UI/issues/1607 to check is it SUI/FUI CSS issue. I will also fix an example.

Thanks for reporting 👍

**Comment by layershifter** [comment]
Thanks for reporting again, I have fixed an example in #3997. The notable change is:

```diff
-<Sidebar.Pushable as={Segment}>
+<Sidebar.Pushable as={Segment} style={{ overflow: 'hidden' }}>
```
